frontend/packages/dev-console/src/components/import/advanced/ProgressiveList.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
```

Comment:
Also, you can move these files out of import folder since its a generic component and will be used by other forms as well. Something like `component/progressive-list` should be fine.